src/PanelTraits/Columns.php

```diff
@@ -88,33 +88,37 @@ public function addColumns($columns)
     /**
      * Moves the recently added column to 'before' the $target_col.
      *
-     * @param $target_col
+     * @param $target string|array the name or array of the target column
      */
-    public function beforeColumn($target_col)
+    public function beforeColumn($target)
```

Comment:
Hey @OliverZiegler, are these fixes merged from other PRs? 
If so, which one? If not, they shouldn't be in the scope of this one.
Since my test branch is based off of this one, I also inherited these changes, which @lloy0076 pointed out.
Because of this, I'm planning to revert them in my PR but I don't know if they're gonna end up merged if I do. Can you provide some context?

Reply:
I have not used this branch anywhere... 
currently too busy to participate in backpack as much as I wanted... I'm sorry for this...